### PR TITLE
fix(eth-staking): fix unstake modal layout, btn disabling etc.

### PR DESCRIPTION
**Pull request by StanCheb**
## Description

The requests (`eth_call`) to public nodes are removed
The "Provided by Everstake" footer is added to the staking tab
The confirm buttons in the stake, unstake and claim modals are now blocked when a device is locked or not available
The confirming tx label text is fixed
The horizontal scrolling in the unstake modal is fixed

## Related Issue

Resolve 
https://github.com/trezor/trezor-suite/issues/9268
https://github.com/trezor/trezor-suite/issues/11414
https://github.com/trezor/trezor-suite/issues/11438

### CODE REVIEW

File: packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EverstakeFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export const EverstakeFooter = () => {
+    const theme = useTheme();
+    const isDarkMode = theme.THEME === 'dark';
+
+    return (
+        <Wrapper>
+            <Left>
+                <Translation id="TR_STAKE_PROVIDED_BY" />{' '}
+                <Image
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EverstakeFooter.tsx]: How about adding just one everstake logo svg and change color using css?

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx [hunk 2]
```diff
@@ -16,9 +16,8 @@ const GreyP = styled(Paragraph)`
 const DividerWrapper = styled.div`
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx:16]: Not sure if related but missing gap under warning 
Screenshot 2024-03-13 at 13 10 48

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EverstakeFooter.tsx (hunk 1 above)
Comment (StanCheb) [reply packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/EverstakeFooter.tsx]: Done. https://github.com/trezor/trezor-suite/pull/11543/commits/08e8109dfc902926c6b8cc6c026729e95801d503

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx (hunk 2 above)
Comment (StanCheb) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx:16]: Done. https://github.com/trezor/trezor-suite/pull/11543/commits/08e8109dfc902926c6b8cc6c026729e95801d503

**Review by tomasklim — APPROVED** [review]

**Pull request closed without merging**